src/ctia/entity/note.clj

```diff
@@ -0,0 +1,121 @@
+(ns ctia.entity.note
+  (:require [ctia.domain.entities :refer [page-with-long-id un-store-page]]
+            [ctia.entity.note.schemas :as note-schemas]
+            [ctia.http.routes.common :as routes.common]
+            [ctia.http.routes.crud :refer [services->entity-crud-routes]]
+            [ctia.lib.compojure.api.core :refer [GET routes]]
+            [ctia.schemas.core :refer [APIHandlerServices]]
+            [ctia.store :refer [list-records]]
+            [ctia.stores.es.mapping :as em]
+            [ctia.stores.es.store :refer [def-es-store]]
+            [schema-tools.core :as st]
+            [schema.core :as s]))
+
+(def note-mapping
+  {"note"
+   {:dynamic false
+    :properties
+    (merge
+     em/base-entity-mapping
+     em/sourcable-entity-mapping
+     em/stored-entity-mapping
+     {:entity_id em/searchable-token
```

Comment:
ids should only user `token` mapping (searchable adds an additional text field to have some fields indexed both as tokens and text, we introduced it for sources to add this additional field without reindexing everything)

Reply:
done